### ISSUE TITLE
C++ and C compiler detection on FreeBSD platform

### ISSUE THREAD

**Issue by GoogleCodeExporter** [waf-project/waf#1528]
```
>> operating system:
FreeBSD 10.10+

On FreeBSD 10+ platforms, the system compiler is clang.  The current 
Tools/compiler_c.py and Tools/compiler_cxx.py do not have special case for 
freebsd platform at all and the default gcc,clang detection is used.

I'm proposing to add special case for freebsd, prioritizing clang compiler over 
gcc.
```

Original issue reported on code.google.com by `caw...@gmail.com` on 22 Jan 2015 at 9:27

**Comment by cawka** [comment]
Just for record.  This issue has been addressed as part of 04449361e8fe2dfe35d3b98c4bc28a991c646a7e

**Comment by ita1024** [comment]
Use waf 1.9.0
